code-quality/accessibility/WCAG-testing.md

```diff
@@ -0,0 +1,102 @@
+# WCAG Testing Procedure
+
+This is the procedure that has been used to test the FUSD WordPress sites
+
+## Automated testing
+
+There are 3 primary tools used in testing, they are all Chrome browser extensions:
+
+- [WAVE Browser Extension for Chrome and Firefox](https://wave.webaim.org/extension/)
+- [SiteImprove Browser Extension](https://siteimprove.com/en-us/core-platform/integrations/browser-extensions/)
+- [Broken Link Checker](https://chrome.google.com/webstore/detail/broken-link-checker/nibppfobembgfmejpjaaeocbogeonhch)
+
+The first two go over the site currently loaded in the browser and point out possible issues. The third will test all links on a site. The first two tests should be done on as many pages as a user is likely to encounter upon a visit. If you can check all of them, great, but if your site has 100 pages, and 95 of those are the result of a template, you don't need to check all 100.
+
+The link checker, on the other hand, should be done on as much of the site as possible to avoid any broken links.
+
+### WAVE
+
+Simply navigate to the website you wish to test and then click the WAVE icon in your browser:
+
+![WAVE Icon illustration](screens/wave-before.png)
+
+This will display several landmark on the screen and a summary on the left side
+
+![WAVE Active on a page](screens/wave-after.png)
+
+Clicking on the "Contrast" tab will display details on any contrast errors in your page. Clicking the flag icon on the left will display the details of errors/alerts on the page:
+
+![WAVE Details](screens/wave-details.png)
+
+This will display any Error and Alerts on the page. Errors should always be fixed. Alerts will can sometimes be ignored. For example, any link to a PDF file will prompt an alert because PDFs can't be tested automatically. Clicking the blue **i** next to the alert will give you a more detailed descripion as to why this is a problem and how it can be remedied.
+
+You don't need to worry about the other categories; the next automated test will present your ARIA landmarks in a way that's easier to evaluate, but this can be helpful to get an overview of the features and landmarks of your page.
```

Comment:
Do we know if HTML5 makes `aria` tagging obsolete?  Or just in some areas?

Reply:
In basically everything I've read HTML5 is preferred over ARIA. I believe this is the reason that SI always asks you about ARIA tags, and if they're applied in conjunction with HTML it will give you a redundant ARIA alert.

Reply:
Ah, interesting.  I guess WP hasn't gotten there yet.

Reply:
As with most things...